library/src/main/java/com/pokegoapi/main/RequestHandler.java

```diff
@@ -56,7 +56,7 @@
 	private final Map<Long, ResultOrException> resultMap = new HashMap<>();
 	private String apiEndpoint;
 	private OkHttpClient client;
-	private Long requestId = new Random().nextLong();
+	private Long requestId = Math.abs(new Random().nextLong());
```

Comment:
LocalThreadRandom has an nextLong method that is accepting an parameter. ThreadLocalRandom.current().nextLong(Long.MAX_VALUE)